run_sets/testsuite.yml

```diff
@@ -1,6 +1,10 @@
+# This file describes the order of features in a normal testsuite run.
+#
+# If you create new features, please see conventions about naming of
+# the feature files in docs/Guidelines.md, as well as guidelines about
```

Comment:
I would add here the chapiter ### Rules for features ( since the file contain other infos)

Reply:
ok